com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameProjectBuilder.java

```diff
@@ -552,23 +421,46 @@ public void build(Task<Void> task) throws CompileExceptionError, IOException {
         IResource input = task.input(0);
 
         BobProjectProperties properties = Project.loadProperties(project, input, project.getPropertyFiles());
+        final String root = FilenameUtils.concat(project.getRootDirectory(), project.getBuildDirectory());
 
         try {
             if (project.option("archive", "false").equals("true")) {
-                ResourceNode rootNode = new ResourceNode("<AnonymousRoot>", "<AnonymousRoot>");
-                List<String> excludedResources = new ArrayList<String>();
-                HashSet<String> resources = findResources(project, rootNode, excludedResources);
-
-
-                ManifestBuilder manifestBuilder = this.prepareManifestBuilder(rootNode, excludedResources);
-
-                // Make sure we don't try to archive the .arci, .arcd, .projectc, .dmanifest, .resourcepack.zip, .public.der
+                // create the resource graphs
+                // the full graph contains all resources in the project
+                // the live update graph contains each resource only once per collection proxy
```

Comment:
it seems like we don't create a separate LU graph anymore?

Reply:
True, we don't. You can remove that comment.